packages/api-reference/src/components/Content/Schema/Schema.vue

```diff
@@ -61,7 +61,7 @@ const handleClick = (e: MouseEvent) =>
       :class="[
         `schema-card--level-${level}`,
         { 'schema-card--compact': compact, 'schema-card--open': open },
-        { 'border-t-1/2': additionalProperties && open },
+        { 'border-t': additionalProperties && open },
```

Comment:
you sure we don’t want the 1/2 border anymore? disclaimer: I know nothing about the borders

Reply:
Our default border size is now `0.5px`. This is legacy from before @cameronrohani changed the default border width.